libia2/include/ia2.h

```diff
@@ -22,10 +22,12 @@
 
 #ifdef LIBIA2_INSECURE
 #define INIT_COMPARTMENT(n)
-#define IA2_WRPKRU
+#define IA2_WRPKRU ""
+#define IA2_RDPKRU ""
 #else
 #define INIT_COMPARTMENT(n) _INIT_COMPARTMENT(n)
 #define IA2_WRPKRU "wrpkru"
+#define IA2_RDPKRU "rdpkru"
```

Comment:
#182 is moving away from having the wrappers depend on ia2.h [(SourceRewriter.cpp:631)](https://github.com/immunant/IA2-Phase2/pull/182/files#diff-ad4877eea1240e8a16c89351f2759f73775a01b7666d3744cad36319e32b7db9R631). If rdpkru is just for this test I'd lean towards having the macro defined in one of the test's headers.

Reply:
nvm I just saw that this is also used in `allocate_compartment_stack`